Framework/PythonInterface/plugins/functions/MsdGauss.py

```diff
@@ -34,7 +34,7 @@ def function1D(self, xvals):
         msd = self.getParameterValue("Msd")
 
         xvals = np.array(xvals)
-        intensity = height * np.exp(-msd * xvals**2)
+        intensity = height * np.exp((-msd * xvals**2)/6)
```

Comment:
This looks like more than a documentation update? Was the formula here incorrect before?

Reply:
Yes the formula was incorrect - I noticed this when updating the MsdGauss documentation. Sanghamitra has informed me that the MSDPeters and MSDYi functions should reduce to the MSDGauss formula given certain conditions, and so this means a 1/6 factor needs to be added to the exponent.

Reply:
:+1: